src/components/GenericError/assets/interrupted-flow.svg

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I hope you handle SVG compression with webpack, or you can use e.g. https://imageoptim.com/mac